bin/check-es-query-ratio.rb

```diff
@@ -253,8 +260,13 @@ def run
     dividend = client.count(build_request_options)
     config[:query] = divisor_query
     divisor = client.count(build_request_options)
+    divisor_zero_ok = config[:divisor_zero_ok]
     if divisor['count'] == 0
-      critical 'Divisor is 0, ratio check cannot be performed, raising an alert'
+      if divisor_zero_ok
```

Comment:
I think this could be more efficiently written to avoid the nested ifs:
```ruby
 if divisor_zero_ok && divisor['count'].zero?
  ok 'Divisor is 0, ratio check cannot be performed, failing safe with ok'
elsif divisor['count'].zero?
  critical 'Divisor is 0, ratio check cannot be performed, raising an alert'
end
```